spec/lucky/text_response_spec.cr

```diff
@@ -30,13 +30,29 @@ describe Lucky::TextResponse do
       context.request.body.to_s.should eq ""
       context.response.status_code.should eq 200
     end
+
+    it "gzips if enabled" do
+      Lucky::Server.temp_config(gzip_enabled: true) do
+        output = IO::Memory.new
+        context = build_context_with_io(output)
+        context.request.headers["Accept-Encoding"] = "gzip"
+        expected_io = IO::Memory.new
+        Gzip::Writer.open(expected_io) { |gzw| gzw.print "some body" }
+
+        print_response(context, status: 200, body: "some body")
+        context.response.close
+
+        context.response.headers["Content-Encoding"].should eq "gzip"
+        output.to_s.ends_with?(expected_io.to_s).should be_true
+      end
+    end
```

Comment:
I think it might be good to have a spec asserting the response is not gzipped if it doesn't have a type supported by gzip. Like if you just did `foo/bar` as the content type it should not be gzipped. What do you think? Would that be worth testing?

Reply:
I agree. I had that spec in the static compression handler, not sure why I didn't add it here. Probably got distracted :)